backend/FwLite/FwLiteShared/Projects/LexboxProjectService.cs

```diff
@@ -16,8 +18,26 @@ public class LexboxProjectService(
     IHttpMessageHandlerFactory httpMessageHandlerFactory,
     BackgroundSyncService backgroundSyncService,
     IOptions<AuthConfig> options,
-    IMemoryCache cache)
+    IMemoryCache cache,
+    GlobalEventBus globalEventBus) : IHostedService
```

Comment:
rather than making this a hosted service we can just subscribe in the constructor and make the class disposable and cleanup the subscription there. I think that would do the same thing